FirstCodingTask/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Women Form</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <Form align="center">
```

Comment:
This should be lower-case, so it should be `<Form>` not `<form>`

Reply:
Noted sir